tutorials/src/androidTest/java/com/kaspersky/kaspresso/tutorials/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.kaspersky.kaspresso.tutorials
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
```

Comment:
```suggestion
 * Instrumented test, which will be executed on an Android device.
```

Reply:
Interesting, I removed this file earlier. Will check it.